### PR TITLE
Updating maintainer

### PR DESCRIPTION
**Pull request by quarkytale**
Adding changes to dashing

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -5,9 +5,9 @@
   <description>
     This rqt plugin provides a way to view and edit parameters on nodes.
   </description>
-  <maintainer email="logans@cottsay.net">Scott K Logan</maintainer>
-  <maintainer email="michael.jeronimo@openrobotics.org">Michael Jeronimo</maintainer>
-
+  
+  <maintainer email="dharini@openrobotics.org">Dharini Dutia</maintainer>
+
```
Comment (clalancette) [inline package.xml]: Nit: this seems to be trailing whitespace, please remove this.

**Review by clalancette — CHANGES_REQUESTED** [review]
One nit, and then this will be good to go.

File: package.xml [hunk 2]
```diff
@@ -5,9 +5,9 @@
...
```
Comment (clalancette) [inline package.xml]: It looks like there is still a trailing space here.

Comment (DLu) [reply package.xml]:
```suggestion

```

**Pull request merged**